### PR TITLE
docs(injector): helper must bind to hexo context

### PR DESCRIPTION
**Pull request by curbengh**
https://hexo.io/api/helper#How-do-I-use-another-registered-helper-inside-my-custom-helper

### CODE REVIEW

File: source/api/injector.md [hunk 1]
```diff
@@ -115,6 +114,7 @@ hexo.extend.injector.register('head_end', require('./lib/inject')(hexo))
 
 ``` js lib/inject.js
 module.exports = (hexo) => () => {
+  const css = hexo.extend.helper.get('css');
```
Comment (curbengh) [inline source/api/injector.md:117]: just tested this line and it doesn't work due to `this.url_for()`.

https://github.com/hexojs/hexo/blob/14890743dd00e1a1e7c4bd9c7ef632f609d35a4c/lib/plugins/helper/css.js#L27

Comment (curbengh) [reply source/api/injector.md:117]: const css = hexo.extend.helper.get('css').bind(hexo) doesn't work for me either, tried v4.0.0, v4.1.1, v4.2.1, v5.

Comment (curbengh) [reply source/api/injector.md]: https://github.com/hexojs/hexo/pull/4447

File: source/api/injector.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ There are other internal functions, see [hexojs/hexo#4049](https://github.com/he
 ## Example
 
 ```js
-const css = hexo.extend.helper.get('css');
+const css = hexo.extend.helper.get('css').bind(hexo);
 const js = hexo.extend.helper.get('js');
```
Comment (jiangtj) [inline source/api/injector.md:50]: https://github.com/hexojs/hexo/blob/14890743dd00e1a1e7c4bd9c7ef632f609d35a4c/lib/plugins/helper/js.js#L32

`js` helper also need bind hexo context,

**Comment by curbengh** [comment]
fix released in [5.0.1](https://github.com/hexojs/hexo/releases/tag/5.0.1)

File: source/api/injector.md (hunk 1 above)
Comment (curbengh) [reply source/api/injector.md]: Fix released in [5.0.1](https://github.com/hexojs/hexo/releases/tag/5.0.1)

**Pull request merged**